content/updates/connectivity/2-Enhanced-connection-monitoring.md

```diff
@@ -0,0 +1,47 @@
+# Enhanced connection monitoring
+
+## Monitor errors on your connections 
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+For each of your connections, a new `Monitoring` page now lists the last integration errors that may have occurred.
+
+From this page, you can now easily track and fix any error on product creation, update, or deletion for each of your connections. 
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+Each warning message shows:
+- a small paragraph that explains what went wrong during the connection's synchronization, 
+- internal links to help you check the data in your PIM,
+- external links to directly access useful information in our help center.
+
+![Enriched error message](../img/fix-errors-on-your-connections.png)
+
+We also enhanced the Connection dashboard with additional information to ease error monitoring and allows you to see at a glance how your source connections are performing.  
+
+To ease error monitoring and allow you to see at a glance how your source connections are performing, the Connection dashboard now displays: 
+- a graph with the number of product errors (business and technical) for the past week,
+- connection cards showing the number of business errors for the past week. 
+
+![Error monitoring on connection dashboard](../img/error-monitoring-connection-dashboard.png)
+
+## Track your destination connections
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/connection-dashboard.html#destination-connection-monitoring"
+:::
+
+You can now easily track the number of products that were sent to your destinations in the Connection dashboard.
+
+You can consult this metric for each of the last 7 days. We also compute the sum of the last 7 days.
+
+![Destination connection chart](../img/number-of-products-sent.png)
+
+## Choose which connection you want to track
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-your-connections.html#enable-the-tracking"
+:::
+
+By default, new connections won't be monitored. If you want to activate their tracking, check the dedicated option in the Connection setting page.
```

Comment:
Maybe add something about the EE version? Because this is valid only for EE customers. Connection are tracked by default for CE customers.

Reply:
Seen with Paul as both Megane is in vacation. Paul confirmed that this feature is for both EE and CE customers.
Maybe were you talking about something else, Megane?